mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Content description (not visible, for screen readers etc.): Description for the overflow menu button in the browser toolbar. -->
+    <string name="mozac_browser_toolbar_menu_button">Menu</string>
+    <string name="mozac_clear_button_description">Clear</string>
```

Comment:
Localizers only see one string at a time, and the comment will only appear the first string.
Please add a comment describing the "Clear" string as well

Reply:
Yes, we are creating another toolbar from copying the original toolbar code.  Can we ignore this for today? The history did not correctly link so I'm going to remove the toolbar2 folder soon.

Reply:
Thanks for the update
I can put these strings on hold for now but as soon as new a-c strings land, they will be part of this PR and so I will be blocked on exposing them.
Do you expect the removal of toolbar2 folder to happen soon?

Reply:
yes, I'm currently removing it today.  One question, if I'm copying the toolbar folder into a new "toolbar-legacy" folder, I should update the clear string?  I can do that for both toolbar and "toolbar-legacy" (not sure the name yet)

Reply:
At the end we're sticking with the current toolbar and toolbar2.  I've put up a fix for [this](https://phabricator.services.mozilla.com/D208425).  I'll land it as soon as I get a r+.